recipes/openssl/3.x.x/test_package/conanfile.py

```diff
@@ -1,40 +1,67 @@
-from conans import CMake, tools, ConanFile
-from conan.tools.build import cross_building
+from conan import ConanFile
+from conan.tools.scm import Version
+from conan.tools.build import can_run
+from conan.tools.cmake import cmake_layout, CMake, CMakeToolchain
 import os
 
+required_conan_version = ">=1.50.2 <1.51.0 || >=1.51.2"
+
 
 class TestPackageConan(ConanFile):
-    settings = "os", "compiler", "arch", "build_type"
-    generators = "cmake", "cmake_find_package", "pkg_config"
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "CMakeDeps", "VirtualRunEnv"
+    test_type = "explicit"
+
+    @property
+    def _skip_test(self):
```

Comment:
You can't skip a test. If something does not work, please, raise a ConanInvalidConfiguration in `validate()` method in the recipe

Reply:
I just copied & pasted this part from the "v2 test package" for 1.x.x: https://github.com/conan-io/conan-center-index/blame/master/recipes/openssl/1.x.x/test_package/conanfile.py#L16

I originates here: https://github.com/conan-io/conan-center-index/pull/9208#issuecomment-1027081580

As this test skipping was not part of the "v1 test package" for 3.x.x, I will give it a try not to have this in the "v2 test package" for 3.x.x